tests/__init__.py

```diff
@@ -78,12 +105,31 @@ def test_b(self):
 
     Regular test cases will be run in usual way, while async ones - inside
     event loop.
+
+    A test class may contain special setup / teardown methods, which will
+    be executed before and after running of each test case::
+
+        class TestSnippets(unittest.TestCase, metaclass=AIOTestMeta):
+
+            def setup(self):
+                self.app = create_app(DEFAULT_CONF)
+
+            def teardown(self):
+                loop = asyncio.get_event_loop()
```

Comment:
Thank you. Not it's good... at least API's good, the implementation makes me a bit scary! ;)